pysatMagVect/_core.py

```diff
@@ -109,10 +109,12 @@ def geodetic_to_ecef(latitude, longitude, altitude):
 
     return x, y, z
 
+import pysatMagVect.fortran_coords
+ecef_to_geodetic = pysatMagVect.fortran_coords.ecef_to_geodetic
 
-def ecef_to_geodetic(x, y, z, method=None):
+def python_ecef_to_geodetic(x, y, z, method=None):
```

Comment:
Curious about this name change

Reply:
I added a ecef_to_geodetic function written in Fortran that I wanted the code to use by default. I kept the python version to check accuracy and it is easier to read.